opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -299,55 +300,36 @@ pub struct BatchConfig {
 
 impl Default for BatchConfig {
     fn default() -> Self {
-        let mut config = BatchConfig {
+        BatchConfigBuilder::default().build()
+    }
+}
+
+/// A builder for creating [`BatchConfig`] instances.
+#[derive(Debug)]
+pub struct BatchConfigBuilder {
+    max_queue_size: usize,
+    scheduled_delay: Duration,
+    max_export_batch_size: usize,
+    max_export_timeout: Duration,
+}
+
+impl Default for BatchConfigBuilder {
+    /// Create a new [`BatchConfigBuilder`] initialized with standard OTLP default values.
+    /// The values are overriden by environment variables if set.
+    /// For a list of supported environment variables see [Batch LogRecord Processor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#batch-logrecord-processor).
+    fn default() -> Self {
+        BatchConfigBuilder {
```

Comment:
Please update the changelog indicating the breaking change that non-standard `OTEL_BLRP_EXPORT_*-MILLIS` env variables are not supported.

Reply:
Ah sorry, please ignore as this was for traces, and it is still supported as I can see :)

Reply:
They are still supported in this PR (see `init_from_env_vars`), but it would make my evening to remove them :)
Should I do it?

Reply:
yes, I would vote for removing these non-standard env-variable if there are no concerns. cc @open-telemetry/rust-maintainers

Reply:
cool, will followup with another PR  to remove those.

Reply:
@lalitb What are the env variables we want to remove?

Reply:
got it. Yes, agree to remove them. Better to separately PR that.